### ISSUE TITLE
handle_frame does not get called on larger payloads delivered via fragments

### ISSUE THREAD

**Issue by weixiyen** [Azolo/websockex#60]
I'm experiencing issues where larger frames (50KB+) delivered via fragments don't complete.

Using trace:

```
*DBG* #PID<0.218.0> received frame: {:fragment, :text ....
*DBG* #PID<0.218.0> received frame: {:continuation, ...
--- gets stuck without more :continuation or :end ----
```

I was wondering if this was a known issue.

Using **elixir-socket** library, I am able to receive every fragment.

**Comment by Azolo** [comment]
It's now a known issue, thanks for reporting it. I'll look into it.

**Comment by Azolo** [comment]
I'm having trouble replicating this, can you provide any other info?

**Comment by rosacris** [comment]
I am evaluating using the library, is this still an issue?

**Comment by Azolo** [comment]
I think this is fixed via #79
